ramalama.py

```diff
@@ -423,20 +29,24 @@ def main(args):
 
         if conman:
             home = os.path.expanduser('~')
-            conman_args = [conman, "run", "--rm", "-it", "--security-opt=label=disable", f"-v{ramalama_store}:/var/lib/ramalama", f"-v{home}:{home}", "-v/tmp:/tmp",
-                           f"-v{__file__}:{__file__}", "-e", "RAMALAMA_HOST", "-p", f"{host}:{port}", "quay.io/ramalama/ramalama:latest", __file__] + args
+            pwd = os.getcwd()
+            conman_args = [conman, "run", "--rm", "-it", "--security-opt=label=disable", f"-v{store}:/var/lib/ramalama", f"-v{home}:{home}", "-v/tmp:/tmp",
+                           f"-v{pwd}:{pwd}", "-e", "RAMALAMA_HOST", "-p", f"{host}:{port}", "quay.io/ramalama/ramalama:latest", __file__] + args
```

Comment:
{pwd}, will only work when executing like this:

./ramalama.py

when calling like this:

ramalama

pwd could be anywhere...

one way to avoid listing all the files individually could be to put all the python files somewhere like:

/usr/libexec/ramalama

and share that whole dir if it exists. I really don't know how imports find the directory to load though... And pwd is useful for development

Reply:
We could mount it at /usr/lib/python3.12/site-packages/ramalama.  If we were to install this for real, then I would have /usr/bin/ramalama and /usr/lib/python3.12/site-packages/ramalama

Reply:
I have a new fix to make this work based on this location, but waiting for this to merge first.